src/components/AssetSearch/AssetSearch.tsx

```diff
@@ -30,7 +30,7 @@ export const AssetSearch = ({ onClick }: AssetSearchProps) => {
 
   const fetchTokens = useCallback(async () => {
     try {
-      const data = await assetService.byNetwork()
+      const data = assetService.byNetwork(NetworkTypes.MAINNET)
```

Comment:
This might be a larger discussion but should we set the network type as a config variable so that we can switch the whole app over from mainnet to test net in one location?

Reply:
Yeah I agree we should add a settings config or user session to handle allowing users to enable/disable mainnet/testnet. It would require unchained testnet support I think so we could change out the chain-adapters